{"name": "Login Authentication, single paramters", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div/div/div[2]/nav/div[3]/h1\"}\n  (Session info: chrome=114.0.5735.248); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x011BA813+48355]\n\t(No symbol) [0x0114C4B1]\n\t(No symbol) [0x01055358]\n\t(No symbol) [0x010809A5]\n\t(No symbol) [0x01080B3B]\n\t(No symbol) [0x010AE232]\n\t(No symbol) [0x0109A784]\n\t(No symbol) [0x010AC922]\n\t(No symbol) [0x0109A536]\n\t(No symbol) [0x010782DC]\n\t(No symbol) [0x010793DD]\n\tGetHandleVerifier [0x0141AABD+2539405]\n\tGetHandleVerifier [0x0145A78F+2800735]\n\tGetHandleVerifier [0x0145456C+2775612]\n\tGetHandleVerifier [0x012451E0+616112]\n\t(No symbol) [0x01155F8C]\n\t(No symbol) [0x01152328]\n\t(No symbol) [0x0115240B]\n\t(No symbol) [0x01144FF7]\n\tBaseThreadInitThunk [0x76B87D59+25]\n\tRtlInitializeExceptionChain [0x77DCB79B+107]\n\tRtlClearBits [0x77DCB71F+191]\n\n", "trace": "  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\loginsteps.py\", line 27, in step_impl\n    text = context.driver.find_element(By.XPATH,\"/html/body/div/div/div[2]/nav/div[3]/h1\").text\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Launch chrome browser", "status": "passed", "start": 1690118820930, "stop": 1690118822728}, {"name": "When Open login page", "status": "passed", "start": 1690118822728, "stop": 1690118824144}, {"name": "And Enter phone \"0208033577\" and pin \"1996\"", "status": "passed", "start": 1690118824144, "stop": 1690118824432}, {"name": "And Click on Enter", "status": "passed", "start": 1690118824432, "stop": 1690118824505}, {"name": "Then successful login", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/div/div/div[2]/nav/div[3]/h1\"}\n  (Session info: chrome=114.0.5735.248); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x011BA813+48355]\n\t(No symbol) [0x0114C4B1]\n\t(No symbol) [0x01055358]\n\t(No symbol) [0x010809A5]\n\t(No symbol) [0x01080B3B]\n\t(No symbol) [0x010AE232]\n\t(No symbol) [0x0109A784]\n\t(No symbol) [0x010AC922]\n\t(No symbol) [0x0109A536]\n\t(No symbol) [0x010782DC]\n\t(No symbol) [0x010793DD]\n\tGetHandleVerifier [0x0141AABD+2539405]\n\tGetHandleVerifier [0x0145A78F+2800735]\n\tGetHandleVerifier [0x0145456C+2775612]\n\tGetHandleVerifier [0x012451E0+616112]\n\t(No symbol) [0x01155F8C]\n\t(No symbol) [0x01152328]\n\t(No symbol) [0x0115240B]\n\t(No symbol) [0x01144FF7]\n\tBaseThreadInitThunk [0x76B87D59+25]\n\tRtlInitializeExceptionChain [0x77DCB79B+107]\n\tRtlClearBits [0x77DCB71F+191]\n\n", "trace": "  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\loginsteps.py\", line 27, in step_impl\n    text = context.driver.find_element(By.XPATH,\"/html/body/div/div/div[2]/nav/div[3]/h1\").text\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 740, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\innoson\\PycharmProjects\\test-oya\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1690118824506, "stop": 1690118824529}], "start": 1690118820929, "stop": 1690118824529, "uuid": "e586dfdf-ea83-4777-9867-b86dc9c0ec9f", "historyId": "2b075725602fad945d7ecdc80b520fba", "fullName": "Login: Login Authentication, single paramters", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}